withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -684,19 +730,19 @@ PROCEDURE VALIDATESETERROR
    IF VALIDATESTATUS=VALIDATEACCTNOTFOUND THEN
     BNOERRMESSAGE="Account Not Found"
    ELSE IF VALIDATESTATUS=VALIDATEWARNINGFOUND THEN
-    BNOERRMESSAGE="Account Warning Found"
+    BNOERRMESSAGE="Account Warning Found List "+INVACCTWARNSFOUND
```

Comment:
I don't think you want to change the main error message.  I would not append " List" to this.  This is still the "Account Warning Found" error.  Instead, I think the pattern is to use the modifier to include the list of warning you actually found.  Maybe something like:
```
Account Warning Found: 001,005,010
```

Reply:
made suggested change